### PR TITLE
 JDK 11 preparation and jakarta uptake. 

### PR DESCRIPTION
**Pull request by Tomas-Kraus**

### CODE REVIEW

File: api/pom.xml [hunk 1]
```diff
@@ -23,12 +23,12 @@
...
         <groupId>org.eclipse.ee4j</groupId>
         <artifactId>project</artifactId>
-        <version>1.0.1</version>
+        <version>1.0.4</version>
     </parent>
 
     <groupId>jakarta.json.bind</groupId>
     <artifactId>jakarta.json.bind-api</artifactId>
-    <version>1.1.0-SNAPSHOT</version>
+    <version>1.0.1-SNAPSHOT</version>
```
Comment (lukasj) [inline api/pom.xml]: you're in master, should stay 1.1.0-SNAPSHOT...

File: api/pom.xml [hunk 2]
```diff
@@ -237,13 +247,17 @@
     </profiles>
 
     <build>
-        <finalName>${project.artifactId}</finalName>
+        <!--finalName>${project.artifactId}</finalName-->
```
Comment (lukasj) [inline api/pom.xml]: remove

File: api/pom.xml (hunk 1 above)
Comment (Tomas-Kraus) [reply api/pom.xml]: Yes, I missed this.

File: api/pom.xml (hunk 2 above)
Comment (Tomas-Kraus) [reply api/pom.xml]: Remved. All projects use default names with versions. I see no reason why this one should have an exception.

File: api/pom.xml [hunk 3]
```diff
@@ -133,7 +147,6 @@
                         <artifactId>maven-javadoc-plugin</artifactId>
                         <configuration>
```
Comment (lukasj) [inline api/pom.xml]: can be completely removed

Comment (Tomas-Kraus) [reply api/pom.xml]: Done

File: api/pom.xml [hunk 4]
```diff
@@ -93,48 +93,58 @@
...
             <id>jdk9</id>
             <activation>
-                <jdk>[1.9,)</jdk>
+                <jdk>[9,)</jdk>
             </activation>
             <build>
                 <plugins>
                     <plugin>
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-javadoc-plugin</artifactId>
```
Comment (lukasj) [inline api/pom.xml]: why are you keeping empty definition here?

Comment (Tomas-Kraus) [reply api/pom.xml]: It was not me who added it. :)
It's gone.

**Pull request merged**